New Audio beta capability (#1500)
saucelabs/sauce-docs: docs/mobile-apps/features/audio-capture.md

### Audio Streaming

<p><small><span className="sauceDBlue">Real Devices Only</span></small></p>

You will be able to stream audio during a Live Testing session in our BETA program! Please contact your account team at Sauce Labs, and we will get you enrolled! 

> This ensures a better digital experience by validating audio playback and testing user flows that require an early audio testing in the development lifecycle.

Reviewer: ```suggestion
It ensures a better digital experience by validating audio playback and testing user flows that require audio testing in the early stages of the development lifecycle.
```

> This helps you to ensure better digital experiences by validating audio playback and testing user flows that require audio testing earlier in the development lifecycle.

Reviewer: ```suggestion
This ensures a better digital experience by validating audio playback and testing user flows that require an early audio testing in the development lifecycle.
```

With the Audio Streaming feature, you are able to hear the sound your application is making during a Live Testing session. It allows you to validate audio streams, notification sounds, and animations. 
You will be able to mute and unmute the sound during your session by clicking on the mute/unmute button on the toolbar.

> After you have been enrolled to the experiment you just need to launch a Live Testing session on a real device, and the audio will be streamed automatically. 

Reviewer: ```suggestion
After you have been enrolled in the BETA program, you just need to launch a Live Testing session on a real device, and the audio will be streamed automatically. 
```

## Limitations

> :::note Not Supported

Reviewer: ```suggestion
:::note Limitations
```
